src/dropdowns/dropdown.ts

```diff
@@ -0,0 +1,209 @@
+import {Directive, Input, Output, EventEmitter, HostBinding, HostListener, ElementRef, OnInit, AfterContentInit, OnDestroy, ContentChildren, QueryList} from 'angular2/core';
+import {NglDropdownTrigger} from './dropdown-trigger';
+import {NglDropdownItem} from './dropdown-item';
+import {NglPageClickService} from './page-click.service';
+import {toBoolean} from '../util/util';
+
+const openEventEmitter: EventEmitter<any> = new EventEmitter<any>(false);
+
+@Directive({
+  selector: '[nglDropdown]',
+  host: {
+    'class': 'slds-dropdown-trigger--click',
```

Comment:
As per https://www.lightningdesignsystem.com/components/menus/ this goes further along the structure, not in the container element.